tests/integration/api/test_decision_environment.py

```diff
@@ -92,6 +92,55 @@ def test_create_decision_environment(
         assert status_message in response.data["eda_credential_id"]
 
 
+@pytest.mark.parametrize(
+    ("credential_inputs", "status_code", "status_message"),
+    [
+        (
+            {"host": "quay.io"},
+            status.HTTP_400_BAD_REQUEST,
+            "Need username and password or just token in credential",
+        ),
+        (
+            {"host": "quay.io", "password": "token1"},
+            status.HTTP_201_CREATED,
+            "",
+        ),
+    ],
+)
```

Comment:
Add a set to test username w/o password.  Result should be status.HTTP_400_BAD_REQUEST.

Reply:
Done